### PR TITLE
Changed update button

### PR DESCRIPTION
**Pull request by arsalan2243**
## Description

change of update button color to have a better contrast in relation to cancel button.

#### Video Link

https://www.loom.com/share/2ba9c5330efd4385a2f6cf46f47af322

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by maidiance — CHANGES_REQUESTED** [review]
This does not look like it's following the style guidelines of the project. Please speak to Swapnali and make sure these changes are approved.

**Comment by fletchulence** [comment]
> 

Feels like this could be a quick fix, did you check with Swapnali about the styleguide?

**Review by fletchulence — CHANGES_REQUESTED** [review]
Black/white are usually not a style preferred by UX. A bit too hard on the eyes from what i understand. Talk to Swapnali and ask her opinion about it - im sure she will be happy to help!

**Review by ChristianSmith2394 — CHANGES_REQUESTED** [review]
As the others have said, not a huge fan of the black and white buttons. They're hard on the eyes. I like the transition into the blue, but think the buttons would just be better off being blue from the start to match the rest of the UI

File: src/components/pages/PendingApplications/PendingApplication.js [hunk 1]
```diff
@@ -74,6 +74,18 @@ const PendingApplications = () => {
                 Date(row.created_at.slice).slice(10, 16),
               button: (
                 <Button
+                  style={{
+                    backgroundImage:
+                      'linear-gradient(-180deg, #37AEE2 0%, #1E96C8 100%)',
+                    borderRadius: '.5rem',
+                    boxSizing: 'border-box',
+                    color: '#FFFFFF',
```
Comment (xokeixo) [inline src/components/pages/PendingApplications/PendingApplication.js:82]: Black button on a blue gradient background? I feel as though that'll be super hard to read for anyone. Maybe something a little better on the eye?

**Pull request closed without merging**